_posts/2018-06-25-java-releases.md

```diff
@@ -159,26 +162,29 @@ important than before to update to a current Java version.
 ## Is Oracle our last hope?
 Until now all strategies and solutions to handle future Java versions are based on the new release train of Oracle. While
 a large majority of all installed JDKs is coming from Oracle there are several other vendors on the market that provide
-a JDK and a JVM. All this products have 1 important thing in common: They all are built on top of OpenJDK. 
+a JDK and a JVM. All this products have one important thing in common: They all are built on top of OpenJDK. 
```

Comment:
I prefer to use real numbers for small values like 1 - 9. Can you please revert this small change.

Reply:
that is contrary to waht writing guidelines suggest: use words up to ten and digits above that ;-)

Reply:
done. I missread the one as I when I first read it. That is why I changed it :wink: